oval-schemas/independent-definitions-schema.xsd

```diff
@@ -968,6 +968,171 @@
+                                                <xsd:appinfo>
+                                                      <sch:pattern id="ind-def_txt54objpattern">
+                                                            <sch:rule context="ind-def:textfilecontent54_state/ind-def:pattern">
+                                                                  <sch:assert test="@operation='pattern match'"><sch:value-of select="../@id" /> - operation attribute for the pattern entity of a shellcommand_object should be 'pattern match'</sch:assert>
+                                                            </sch:rule>
+                                                      </sch:pattern>
+                                                </xsd:appinfo>
+                                          </xsd:annotation>
+                                    </xsd:element>
+			                        <xsd:element name="exit_status" type="oval-def:EntityStateIntType" minOccurs="0" maxOccurs="1">
+										<xsd:annotation>
+											<xsd:documentation>The 'exit_status' entity represents the exist status returned by the system for the execution of the object command.</xsd:documentation>
+										</xsd:annotation>
+									</xsd:element>
+                                    <xsd:element name="stdout_line" type="oval-def:EntityStateAnySimpleType" minOccurs="0" maxOccurs="unbounded">
+                                          <xsd:annotation>
+                                                <xsd:documentation>The 'stdout_line' entity represents a line from the STDOUT output of a successful run of the command string that matched the specified object pattern.</xsd:documentation>
+                                          </xsd:annotation>
+                                    </xsd:element>
+                                    <xsd:element name="subexpression" type="oval-def:EntityStateAnySimpleType" minOccurs="0" maxOccurs="1">
+                                          <xsd:annotation>
+                                                <xsd:documentation>The subexpression entity represents a value to test against the subexpression in the specified pattern. If multiple subexpressions are specified in the pattern, this value is tested against all of them. For example, if the pattern abc(.*)mno(.*)xyp was supplied, and the state specifies a subexpression value of enabled, then the test would check that both (or at least one, none, etc. depending on the entity_check attribute) of the subexpressions have a value of enabled.</xsd:documentation>
+                                          </xsd:annotation>
+                                    </xsd:element>
+                                    <xsd:element name="stderr_line" type="oval-def:EntityStateAnySimpleType" minOccurs="0" maxOccurs="unbounded">
+                                          <xsd:annotation>		
```

Comment:
Should probably be an EntityStateStringType, since the error output is pretty much impossible to control.

Reply:
agreed.  I'll change this.